fairlearn/metrics/_exposure.py

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+from typing import Any
+
+from fairlearn.metrics._input_manipulations import _convert_to_ndarray_and_squeeze
+from ._metric_frame import MetricFrame
+
+_EMPTY_INPUT_PREDICTIONS_ERROR_MESSAGE = "Empty {0} passed to exposure function."
+_INVALID_RANKING_ERROR_MESSAGE = "Please input a valid complete ranking."
+
+
+def exposure(y_true,
+             y_pred,
+             *,
+             pos_label: Any = 1,
+             sample_weight=None) -> float:
+    """Calculate the exposure allocated to the ranking in y_pred.
+
+    For consistency with other metric functions, the ``y_true`` and ``pos_label`` arguments
```

Comment:
Would it break something to remove them? As in, is `pos_label` really there in every metric?